packages/api-db-mongodb/src/db/user.ts

```diff
@@ -44,7 +48,8 @@ export class MongoDBUserAdapter implements DBUserAdapter {
         email: input.email,
         name: input.name,
         roleIDs: input.roleIDs,
-        password: passwordHash
+        password: passwordHash,
+        paymentProviderCustomers: {}
```

Comment:
What is it for? O.o

Reply:
After a successful payment from stripe or other providers we can save a customer with the payment information. This can be later used for payments without user interaction.

Reply:
Ok you mean we will save the payment information so if there is a requiring transaction, we can use it?